docs/topics/braintree.rst

```diff
@@ -3,4 +3,23 @@
 Braintree
 #########
 
-More to come.
+Generate a token
+----------------
+
+Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate>`_:
+
+.. http:post:: /braintree/token/generate/
+
+    **Request**
```

Comment:
I feel like we're going to need to pass in ``customer_id`` but we could always add that later. I'm thinking we'll want it when dealing with saved customer cards.